doc/releases/changelog-0.38.0.md

```diff
@@ -6,41 +6,317 @@
+
+  print(qml.draw(circuit)())
+  ```
+
+  ```pycon
+  0: ──H────────╭●─┤     
+  1: ──H────────│──┤     
+  2: ──H────────│──┤     
+  3: ──RX(0.20)─│──┤  <Y>
+  4: ──RX(0.20)─│──┤  <Y>
+  5: ───────────╰X─┤  
+  ```
+
+  In tandem with `qml.registers`, we've also made the following improvements to `qml.wires.Wires`:
+
+  * `Wires` instances now have a more copy-paste friendly representation when printed.
+    [(#5958)](https://github.com/PennyLaneAI/pennylane/pull/5958)
+
+    ```python
+    >>> from pennylane.wires import Wires
+    >>> w = Wires([1, 2, 3])
+    >>> w
+    Wires([1, 2, 3])
+    ```
+
+  * Python set-based combinations are now supported by `Wires`.
+    [(#5983)](https://github.com/PennyLaneAI/pennylane/pull/5983)
+
+    This new feature unlocks the ability to combine `Wires` instances in the following ways:
+
+    * intersection with `&` or `intersection()`: 
+
+      ```python
+      >>> wires1 = Wires([1, 2, 3])
+      >>> wires2 = Wires([2, 3, 4])
+      >>> wires1.intersection(wires2) # or wires1 & wires2
+      Wires([2, 3])
+      ```
+
+    * symmetric difference with `^` or `symmetric_difference()`:
+
+      ```python
+      >>> wires1.symmetric_difference(wires2) # or wires1 ^ wires2
+      Wires([1, 4])
+      ```
+
+    * union with `|` or `union()`:
+
+      ```python
+      >>> wires1.union(wires2) # or wires1 | wires2
+      Wires([1, 2, 3, 4])
+      ```
+
+    * difference with `-` or `difference()`:
+
+      ```python
+      >>> wires1.difference(wires2) # or wires1 - wires2
+      Wires([1])
+      ```
+
+<h4>Quantum arithmetic operations 🧮</h4>
+
+* Several new operator templates have been added to PennyLane that let you perform quantum arithmetic 
+  operations.
+
+  * In-place operations: :math:`\texttt{SomeOp}(k, mod) \vert x \rangle = \vert x * k \texttt{modulo} mod \rangle` 
+    where :math:`*` denotes addition (:math:`+`) or multiplication :math:(`\times`). Each of the following 
+    operations has the same call signature: `SomeOp(k, x_wires, mod, work_wires=None)`, where `x_wires` 
+    denotes the wires we operate on, :math:`\texttt{mod} = 2^{\texttt{len(x_wires)}}` by default, and
+    `work_wires` are auxiliary wires that may be required to perform the operation.
+  
+    * `qml.Adder` performs in-place modular addition: 
+      :math:`\texttt{Adder}(k, mod)\vert x \rangle = \vert x + k \texttt{modulo} mod \rangle`. 
+      [(#6109)](https://github.com/PennyLaneAI/pennylane/pull/6109)
+
+      ```python
+      dev = qml.device("default.qubit", shots=1)
+
+      @qml.qnode(dev)
+      def circuit():
+          qml.X(0) # |2> = |10> state
+          qml.Adder(1, x_wires = [0, 1])
+          # add 1 
+          # 2 + 1 = 3, or |11> 
+          return qml.sample(wires=[0, 1])
+      ```
+
+      ```pycon
+      >>> circuit()
+      tensor([1, 1], requires_grad=True)
+      ```
+
+    * `qml.PhaseAdder` is similar to `qml.Adder`, but it performs in-place modular addition in the Fourier 
```

Comment:
Not married to having a code example for `qml.PhaseAdder`

Reply:
I would say you can remove code examples from all. Just have a bullet point for each function description, and then have a single, general code example at the end.

Reply:
Yeah, I'm in favour of that @josh146. Too verbose at the moment 😅

Reply:
See new commits!